src/gitlab-ci.coffee

```diff
@@ -132,6 +135,47 @@ module.exports = (robot) ->
             return
 
         return "#{major}.#{minor}.#{dev}"
+    triggerBuildPost = (res, updateRichmond, quickly, branch, buildTagPrefix) ->
+        triggerURL = 'https://agra.sdelements.com/api/v4/projects/33/trigger/pipeline/'
```

Comment:
@mroote @geoffwhittington I updated the code, please comment on the new code if you still think it needs to be changed.